divi/src/main.cpp

```diff
@@ -2733,6 +2840,11 @@ static bool SetPeerVersionAndServices(CNode* pfrom, CAddrMan& addrman, CDataStre
         return false;
     }
 
+    ChainstateManager chainstate;
+    const auto& coinsTip = chainstate.CoinsTip();
+    const auto& blockMap = chainstate.GetBlockMap();
+    const auto& chain = chainstate.ActiveChain();
+
```

Comment:
This seems like this could be removed. It doesn't look like anything is using this

Reply:
Indeed!